### PR TITLE
Scroll to "show test networks" setting from network dropdown link

### PR DESCRIPTION
**Pull request by ryanml**
https://user-images.githubusercontent.com/8732757/141830252-982454b4-66f0-4f03-a07d-21bf53c94846.mov

### CODE REVIEW

**Review by danjm — APPROVED** [review]

**Comment by danjm** [comment]
Looks good!

File: ui/pages/settings/advanced-tab/advanced-tab.component.js [hunk 1]
```diff
@@ -61,6 +61,14 @@ export default class AdvancedTab extends PureComponent {
     showLedgerTransportWarning: false,
   };
 
+  showTestNetworksRef = React.createRef();
+
+  componentDidMount() {
+    if (window.location.hash.match(/show-testnets/u)) {
+      this.showTestNetworksRef.current.scrollIntoView({ behavior: 'smooth' });
```
Comment (Gudahtt) [inline ui/pages/settings/advanced-tab/advanced-tab.component.js:68]: We should probably focus this element as well, for keyboard users

Comment (danjm) [reply ui/pages/settings/advanced-tab/advanced-tab.component.js:68]: ah, nice catch

**Review by danjm — CHANGES_REQUESTED** [review]
Mark made a good suggestion

**Review by danjm — DISMISSED** [review]

**Review by danjm — APPROVED** [review]

**Review by tmashuang — APPROVED** [review]
Approving, but since we are on the discussion for keyboard accessibility and to clarify, none of our toggles are keyboard accessible. I am unable to switch them on/off with my keyboard on macos.

**Review by jpuri — APPROVED** [review]

**Pull request merged**